doc/api/fs.md

```diff
@@ -1308,6 +1308,38 @@ characters directly to the `prefix` string. For instance, given a directory
 `prefix` must end with a trailing platform-specific path separator
 (`require('node:path').sep`).
 
+### `fsPromises.mkdtempDisposable(prefix[, options])`
+
+<!-- YAML
+added: TODO
+changes:
+  - version:
+      - TODO
+    pr-url: TODO
+    description: Initial version.  <-- this doesn't seem to be common practice but the result is that there is no pr-url for the initial feature
```

Comment:
It would need to be properly addressed in the tooling, ideally without duplicating the info
```suggestion
```